website/pages/en/deploying/deploying-a-subgraph-to-studio.mdx

```diff
@@ -25,44 +44,103 @@ yarn global add @graphprotocol/graph-cli
 npm install -g @graphprotocol/graph-cli
 ```
 
-## Create your Subgraph in Subgraph Studio
+## Create your Subgraph
+
+### Create an Account
```

Comment:
This heading doesn't seem super relevant; I would expect the steps below to go through the onboarding process including the email verification step, but it only mentions the first step which is to connect the wallet. Furthermore, the sub-headings ("How to Create a Subgraph in Subgraph Studio" and "Subgraph Compatibility with The Graph Network") are not about creating an account but rather about creating a subgraph, so I'm not sure if you meant to make them sub-headings or if they should be at the same level as this one (`###`)? Even Marcus's video doesn't go through the actual "create an account" process since he logs in to an existing account with multiple subgraphs.